e2e/scenario_test.go

```diff
@@ -694,7 +694,7 @@ func Test_Ubuntu2204Gen2ContainerdAirgapped_K8sNotCached(t *testing.T) {
 		},
 		Config: Config{
 			Cluster: ClusterKubenetAirgap,
-			VHD:     config.VHDUbuntu2204Gen2ContainerdAirgapped,
+			VHD:     config.VHDUbuntu2204Gen2ContainerdAirgappedK8sNotCached,
```

Comment:
Do we really need this test?
I'd like to see the VHD to go away.

Reply:
from network isolated cluster feature perspective, i may still need such feature. Because we exist customer still use old VHD version (from frontend validator, we only block version < 20240906). So we need new CSE script does not generate new egress traffic for old vhd